### PR TITLE
Allows to send email to multiple recipients.

### PR DESCRIPTION
**Pull request by anjali-92**

### CODE REVIEW

File: apphelpers/utilities/email.py [hunk 1]
```diff
@@ -26,7 +26,7 @@ def send_email(
 
     msg["Subject"] = subject
     msg["From"] = sender
-    msg["To"] = recipient
+    msg["To"] = ', '.join(recipient)
```
Comment (gauravr) [inline apphelpers/utilities/email.py:29]: @anjali-92 please rename `recipient` => `recipients`

File: apphelpers/utilities/email.py [hunk 2]
```diff
@@ -44,6 +44,6 @@ def send_email(
     if settings.MD_USERNAME:
         s.login(settings.MD_USERNAME, settings.MD_KEY)
 
-    s.sendmail(msg["From"], msg["To"], msg.as_string())
+    s.sendmail(msg["From"], recipient, msg.as_string())
```
Comment (gauravr) [inline apphelpers/utilities/email.py:47]: @anjali-92 `s.sendmail(sender, recipients, msg.as_string())`

**Review by gauravr — APPROVED** [review]

File: apphelpers/utilities/email.py [hunk 3]
```diff
@@ -26,7 +26,7 @@ def send_email(
 
     msg["Subject"] = subject
     msg["From"] = sender
-    msg["To"] = recipient
+    msg["To"] = ', '.join(recipients)
```
Comment (shon) [inline apphelpers/utilities/email.py:29]: Check https://docs.python.org/3/library/smtplib.html#smtplib.SMTP.sendmail

> to_addrs: a **list** of RFC 822 to-address strings (a bare string will be treated as a list with 1 address)

So possibly we don't need string conversion here.
cc: @gauravr

Comment (gauravr) [reply apphelpers/utilities/email.py:29]: @shon List works at the time of calling `s.sendmail(sender, recipients, msg.as_string())`.
But for building msg, we need to pass string.

**Pull request merged**